manifest-build-tools/lib/RepositoryOperator.py

```diff
@@ -108,7 +108,7 @@ def do_one_task(self, name, data, results):
         results['commands'] = []
         commands = results['commands']
         git = GitBit(verbose=False)
-        if 'credentials' in data:
+        if 'credentials' in data and data['credentials'] is not None:
```

Comment:
:-1: 
conflict with Tian's PR https://github.com/RackHD/on-tools/pull/59/files, which is more completed.
please align with it